docs/kbs_attestation_protocol.md

```diff
@@ -383,20 +383,22 @@ A request for protected resource can fail for three reasons:
 3. The requested resource does not exist. The KBS implementation sends an HTTP
    response with a 404 (`Not Found`) status code.
 
-The KBS protocol currently supports two kinds of resources for an attester to
-request: keys and tokens.
+### Resource Path
```

Comment:
Could we call that one `### Secret Resource`

Reply:
Sure, updated now.